test/units/testsuite-25.sh

```diff
@@ -120,6 +120,7 @@ test ! -f /var/lib/machines/scratch4
 machinectl image-status scratch4 && { echo 'unexpected success'; exit 1; }
 
 # Test import-tar hyphen/stdin pipe behavior
+# shellcheck disable=SC2002
 cat /var/tmp/scratch.tar.gz | machinectl import-tar - scratch5
```

Comment:
`machinectl import-tar - scratch5 < /var/tmp/scratch.tar.gz` or I am missing something?

Reply:
This explicitly tests behavior of `machinectl import-tar -` in combination with pipe, so the pipe here is intentional.

Reply:
Do you mean it is testing reading from a file descriptor connected to a pipe rather than from a file descriptor connected to a file? Maybe this should be mentioned more explicitly in the comment. It would also make sense to mention which check the shellcheck annotations disable in a comment.